src/renderer/scss/_gui.scss

```diff
@@ -1,86 +1,52 @@
-@import url(https://fonts.googleapis.com/css?family=Roboto:400,400i,500,500i,700);
+// Generic html styles used accross the App
+// component specific styling should go in the component scss file
 
-html {
-  height: 100%;
-  font-size: var(--font-size);
+// The actual fonts used will change ex: medium vs regular
```

Comment:
This started as the equivalent of an "everything" CSS file and likely contains some of the worst CSS in the project. Please do your best to refactor as you work on this.

Reply:
Yeah I plan to get rid of or move all of the specific styling. Then keep this is the global html/app-wide styling